flask_request_validator/validator.py

```diff
@@ -188,13 +192,13 @@ def __all_params_are_of_type(params, param_type: str) -> bool:
     return True
 
 
-def __check_if_too_much_params_in_request(params):
+def __check_if_too_many_params_in_request(params):
     expected = {param.name for param in params}
     actual = request.get_json().keys()
     unexpected_keys = {key for key in actual if key not in expected}
 
     if unexpected_keys:
-        raise TooMuchArguments(f'Got unexpected keys: {unexpected_keys}')
+        raise TooManyArguments(f'Got unexpected keys: {unexpected_keys}')
```

Comment:
I apologize. Didn't notice it before (
We can't change exception type in `path` version. It will be incompatible

Reply:
I think you are wrong here. My code is completly backward compatible to 3.0.0 since I use an alias for the exception.
Look at this test: https://github.com/d-ganchar/flask_request_validator/pull/51/commits/fb5f95909be7aaa5874e3ceb1600ba3d25a363c7

Reply:
@LostInDarkMath oh yeah! my mistake. sorry